MonMaraicher/Features/FarmerDetail/FarmerDetailsView.swift

```diff
@@ -130,18 +147,43 @@ private extension FarmerDetailsView {
                     .background(.ultraThinMaterial)
                     .clipShape(.capsule)
                     .padding()
-                    .shadow(radius: 2)
+                    .shadow(radius: 4)
             }
             Divider()
-            Text("Téléphone")
-                .foregroundStyle(.secondary)
-                .font(.subheadline)
-            Text(viewModel.phoneNumber ?? "")
-                .font(.callout)
+            HStack {
+                VStack(alignment: .leading) {
+                    Text("Téléphone")
+                        .foregroundStyle(.secondary)
+                        .font(.subheadline)
+                    Text(viewModel.phoneNumber ?? "Aucun numéro disponible")
+                        .font(.callout)
+                }
+                Spacer()
+                phoneButton
+                    .frame(width: 80, height: 35)
+                    .background(.ultraThinMaterial)
+                    .clipShape(.capsule)
+                    .padding()
+                    .shadow(radius: 4)
+            }
             Divider()
-            Text("Site Web")
-                .foregroundStyle(.secondary)
-                .font(.subheadline)
+            VStack(alignment: .leading) {
+                Text("Email")
+                    .foregroundStyle(.secondary)
+                    .font(.subheadline)
+                Link(viewModel.email!, destination: URL(string: viewModel.email!)!)
+            }
+            Divider()
+            VStack(alignment: .leading) {
+                if !viewModel.websites.isEmpty {
+                    Text("Site Web")
+                        .foregroundStyle(.secondary)
+                        .font(.subheadline)
+                    ForEach(viewModel.websites, id: \.id) { website in
+                        Link(website.websiteType.name, destination: URL(string: website.url)!)
```

Comment:
You should avoid implicit unwrap here : `URL(string: website.url)!`
If the API returns a wrong URL your app will crash